### PR TITLE
remove special case for streaming

### PR DESCRIPTION
**Pull request by dx-pbuckley**

### CODE REVIEW

File: map_configs/dx_map_config.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ push:
 
     - match:
         owner: dataxu
-        repo: "edge|streaming"
+        repo: "edge"
         branch: "master-aws|rel-aws|rel-ec2-master"
```
Comment (cchristous) [inline map_configs/dx_map_config.yml]: also remove `rel-aws`

**Comment by dx-pbuckley** [comment]
@cchristous I've removed `rel-aws` as well, you're confident edge doesn't use it?

**Comment by cchristous** [comment]
edge uses `rel-ec2-master`

**Comment by ferrants** [comment]
looks good to me

**Comment by rdialani** [comment]
@dx-pbuckley 
The above fix still did not lead to auto trigger of streaming-master build today. Can you please check.
Commit Id : f0a0a59a569e3f3586057ab29e98e2f111e147d8

/cc @dxclee

**Comment by dx-pbuckley** [comment]
@ferrants could you look at this today or tomorrow since I'm out? Seems like github-webhooks thinks everything is fine. I logged in and confirmed that the dx_map_config looks correct on the docker container, and here's an example event that reports success (but it didn't really trigger streaming_master):
```
{"action":"triggered streaming_master job","branch":"master","hash":"f0a0a59a569e3f3586057ab29e98e2f111e147d8","issue_number":null,"issue_url":null,"job_name":"streaming_master","label":null,"message":"Merge pull request #383 from dataxu/dev-ut-failure-mac\n\nUpdate testUniqueIds() test to not fail on Mac OS","pr_number":null,"pr_url":null,"pusher":"rdialani","repo":"streaming","repo_owner":"dataxu","time":1456365950267,"type":"push"}
```

**Comment by ferrants** [comment]
the issue is that the streaming_master job has a parameter. fixing in a pr

**Comment by ferrants** [comment]
testing https://github.com/dataxu/jenkins-jobs/pull/1244

**Comment by ferrants** [comment]
confirmed fixed: http://jenkins.devaws.dataxu.net/job/streaming_master/259/

**Comment by cchristous** [comment]
:shipit:

**Pull request merged**